test/clt-tests/sharding/base/config/searchd-with-flexible-ports.conf

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+cat << EOF
+common {
+	plugin_dir = /usr/local/lib/manticore
+	lemmatizer_base = /usr/share/manticore/morph/
+}
+searchd {
+	listen = ${INSTANCE}306:mysql41
+	listen = ${INSTANCE}312
+	listen = ${INSTANCE}308:http
+	log = /var/log/manticore-${INSTANCE}/searchd.log
+	query_log = /var/log/manticore-${INSTANCE}/query.log
+	pid_file = /var/log/manticore-${INSTANCE}/searchd.pid
+	data_dir = /var/log/manticore-${INSTANCE}
+	query_log_format = sphinxql
+	query_log_commands = 1
+	buddy_path = manticore-executor /workdir/src/main.php
```

Comment:
It's good idea to extract it to some env variable like ${EXTRA_SECTION} and use to assign only in test when we need it, otherwise DO not replace Buddy

Reply:
Done